.travis.yml

```diff
@@ -2,13 +2,11 @@ language: python
 
 matrix:
   include:
-  - python: "2.7"
```

Comment:
I think it's better to keep 2.7 until we officially drop the support.

Reply:
Done. Sorry about that, I'll be more accurate next time :)